tcms/static/js/lib/moment-timezone-with-data.js

```diff
@@ -0,0 +1,1540 @@
+//! moment-timezone.js
```

Comment:
This is already available under node_modules, use it

Reply:
no, it isn't.
```
$ ls node_modules/moment-timezone/
LICENSE                  builds                   composer.json            index.js                 moment-timezone.js
README.md                changelog.md             data                     moment-timezone-utils.js package.json
```
the `data` folder contains the data in a .json file, but we still need to load it somewhere and the easier way to do this is to vendor this file